clang/lib/CodeGen/CGExprScalar.cpp

```diff
@@ -197,6 +197,18 @@ static bool CanElideOverflowCheck(const ASTContext &Ctx, const BinOpInfo &Op) {
   if (!Op.mayHaveIntegerOverflow())
     return true;
 
+  if (Op.Ty->isSignedIntegerType() &&
```

Comment:
Does this (and the following if-then) need a comment? Or is it covered by the comment above?

Reply:
I'll add a comment, good idea :)